src/utils/color-utils.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-cond-assign */
+import CSSColors from './css-colors';
+
+// Resolve color from a palette. We should open Theme API from sense so we can use same functionality
+const colorUtils = {
+  resolveExpression: input => {
+    // rgb
+    let matches = /^rgb\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*\)$/i.exec(input);
+    if (matches) {
+      return `rgb(${matches[1]},${matches[2]},${matches[3]})`;
+    }
+    // argb
+    matches = /^argb\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*\)$/i.exec(input);
+    if (matches) {
+      const a = Math.round((matches[1] / 255) * 100) / 100;
+      return `rgba(${matches[2]},${matches[3]},${matches[4]},${a})`;
+    }
+    // hex
+    matches = /^#([A-f0-9]{2})([A-f0-9]{2})([A-f0-9]{2})$/i.exec(input);
+    if (matches) {
+      return input;
+    }
+    // css color
+    const color = input && CSSColors[input.toLowerCase()];
+    if (color) {
+      const a = color.a !== undefined ? color.a : 1;
+      return `rgba(${color.r},${color.g},${color.b},${a})`;
+    }
```

Comment:
@Caele Max is away for some days, so maybe you can help here. Is there any reason why we need to convert the color to rgba? This is done the same way as in the client, but I would say it would work just putting the color directly in the style as well. 
Could it be that this is for security reasons, to make sure it is an actual color that is set? Still no need to convert to rgba I would say though

Reply:
If we don't this look up the color, there is no way of setting a default color. We return 'none' if there are none of the cases are true. But it could just be a look-up table kind of, returning true if it is a correct color, then not converting it to rgb(a).

Reply:
No, but you can type whetever you want in the expression field. If we don't have the list of colors, we have no way of checking if the color name is correct

Reply:
don't know exactly why we check against a list of colors, could be that we want to be able to set transparency for example. Or being extra careful security wise. Either way, this looks fine I think.